pyproject.toml

```diff
@@ -53,6 +53,7 @@ classifiers = [
 # this set should be kept minimal!
 dependencies = [
   "joblib<1.5,>=1.2.0",  # required for parallel processing
+  "nnetsauce>=0.23.2", 
```

Comment:
As mentioned in your previous PR, we want to minimal core dependencies which are necessary for the entire framework, and estimator specific dependencies should be added in different extras. This line should be removed from `dependencies`, and a similar line should be added in `forecasting` extra instead.

Reply:
Also, it'd make sense if you add it in the form of `nnetsauce<0.24,>=0.23.2`, assuming `nnetsause` is following semantic versioning.

Reply:
Done. Is there anything else missing? 

https://github.com/thierrymoudiki/sktime/blob/main/examples/nnetsauce/2024-11-13-nnetsauce-MTS-example.ipynb